src/components/OperationTag.js

```diff
@@ -40,18 +33,21 @@ export default class OperationTag extends React.Component {
     let isShownKey = ["operations-tag", tag]
     let showTag = layoutSelectors.isShown(isShownKey, docExpansion === "full" || docExpansion === "list")
 
+    const opBlockSectionKeyId = isShownKey.map(v => escapeDeepLinkPath(v)).join("-")
+    const opBlockSectionCollapseKeyId = opBlockSectionKeyId+'-collapse'
     return (
       <div
-        className={showTag ? "opblock-tag-section is-open" : "opblock-tag-section"}
-      >
-        <h1
-          className={!tagDescription ? "opblock-tag no-desc" : "opblock-tag" }
-          id={isShownKey.map(v => escapeDeepLinkPath(v)).join("-")}
+        className={showTag ? "opblock-tag-section" : "opblock-tag-section"}
+        >
+        <button
```

Comment:
Lol I'm so sorry but hear me out...
This still has an accessibility issue because there's the possibility of an anchor link inside of this element.

I think I found a solution using the previous commit - we can change this back to a div with `role="rowheader"` - add the click / key listeners / tabindex back (also probably use the previous `className` value too